x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 gnome2-utils
+
+DESCRIPTION="A GUI for f.lux"
+HOMEPAGE="https://justgetflux.com/"
+SRC_URI="https://github.com/xflux-gui/fluxgui/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
```

Comment:
Is the `-*` strictly necessary on this package?

Reply:
Depends. As it needs x11-misc/xflux, which is only a binary package for amd64 and x86, it doesn't make sense on others arches?